dart/lib/src/sentry_client.dart

```diff
@@ -83,6 +83,16 @@ class SentryClient {
     dynamic stackTrace,
     Hint? hint,
   }) async {
+    if (_options.containsIgnoredExceptionForType(event.throwable)) {
+      _options.logger(
+        SentryLevel.debug,
+        'Event was dropped as the exception ${event.throwable.runtimeType.toString()} is ignored.',
+      );
+
+      _recordLostEvent(event, DiscardReason.eventProcessor);
+      return _sentryId;
```

Comment:
this isn't really part of this PR but I feel like this should be renamed to something like `_emptySentryId`